tutorials/building_a_simple_web_blog_with_vweb/README.md

```diff
@@ -207,7 +209,7 @@ fn main() {
 	}
 	sql app.db {
 		create table Article
-	}
+	} or { panic(err) }
```

Comment:
```suggestion
	}!
```

Reply:
Wouldn't it be `}?` since the return type is `?Article`?  Or should the return type be changed to `!Article`?

Reply:
This is the main function , its return type is `!` 

The one function that return ?Article is not really used anywhere (except an example that I updated)

Reply:
also if you mean that retrieve_article function specifically, the possible sql error is handled inside the function with or panic, and the fact that the article maybe absent is already propagated (I'm assuming via indexing, as in indexing gives a ?value)